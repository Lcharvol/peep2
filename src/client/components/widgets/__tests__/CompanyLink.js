import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme'; // eslint-disable-line
import CompanyLink from '../CompanyLink';

describe('<CompanyLink />', () => {
  test('should render an <EmptySearch />', () => {
    const tree = shallow(<CompanyLink />);
    expect(tree).toMatchSnapshot();
  });
  test('should <CompanyLink /> have text transform', () => {
    const tree = shallow(<CompanyLink />);
    expect(tree).toHaveStyleRule('text-transform', 'capitalize');
  });
});
