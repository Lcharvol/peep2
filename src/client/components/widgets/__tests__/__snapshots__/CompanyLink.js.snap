// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompanyLink /> should render an <EmptySearch /> 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Link
    className="sc-bdVaJa lhUzKz"
    replace={false}
/>,
  "nodes": Array [
    <Link
      className="sc-bdVaJa lhUzKz"
      replace={false}
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <Styled(Link) />,
    "_instance": StyledComponent {
      "attrs": Object {},
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {
        "generatedClassName": "lhUzKz",
        "theme": null,
      },
      "unsubscribeId": -1,
      "updater": Updater {
        "_renderer": [Circular],
      },
    },
    "_newState": Object {
      "generatedClassName": "lhUzKz",
      "theme": null,
    },
    "_rendered": <Link
      className="sc-bdVaJa lhUzKz"
      replace={false}
/>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Styled(Link) />,
}
`;
